### PR TITLE
[furyosquad] add age check

### PR DESCRIPTION
**Pull request by XenonR**
Closes desires 🙃

### CODE REVIEW

**Comment by newaccforhc** [comment]
> Closes desires

Lol... Haha.. Nice choice of words. xD

File: src/web/mjs/connectors/FuryoSquad.mjs [hunk 1]
```diff
@@ -29,10 +29,10 @@ export default class FuryoSquad extends Connector {
     }
 
     async _getChapters(manga) {
-        const request = new Request(new URL(manga.id, this.url), this.requestOptions);
-        const data = await this.fetchDOM(request, 'div.list.fs-chapter-list div.title a');
+        const url = new URL(manga.id, this.url);
+        const data = await this._fetchPOST(url, 'adult=true', 'div.list.fs-chapter-list div.title a');
 
-        return data.map(chapter => {
+        return [...data].map(chapter => {
```
Comment (27136773) [inline src/web/mjs/connectors/FuryoSquad.mjs:35]: May move spread operator into `_fetchPOST`

**Review by 27136773 — APPROVED** [review]
Looks good

File: src/web/mjs/connectors/FuryoSquad.mjs (hunk 1 above)
Comment (XenonR) [reply src/web/mjs/connectors/FuryoSquad.mjs:35]: I'll leave it for now. Still playing with the idea of implementing a `Connector.fetchPOST` which will be cleaner and more generic.

**Pull request merged**